double-numbers-using-map.rb

```diff
@@ -0,0 +1,6 @@
+# Use the map function to double all the elements in the array.
+arr = [2,4,6,8]
+new_arr = arr.map {
+    |a| a * 2
+}
```

Comment:
Use `do..end` syntax for multi-line blocks otherwise write the logic in single line.
 ```suggestion
new_arr = arr.map { |a| a * 2 }
```